File: Case point/With angular/CustomerTokenUi/src/app/shared/models/token.ts
export interface TokenListModel
{
    id:number,
    query:number,
    status:number
}

export interface Token
{
    id:number,
    queryId:number,
    statusId:number,
    phone:string,
    address :string
}


export interface ResolveToken
{
    id:number,
    query:number,
    phoneNumber:string,
    address :string
}


export interface TokenCreateModel
{
    query:number
}

File: Case point/With angular/CustoomerToken/CustoomerToken/Domain/Tokens/Token.cs
using CustoomerToken.Domain.Tokens;

public class Token
{
    public int Id { get; set; }
    public QueryType QueryId { get; set; }
    public QueryStatus StatusId { get; set; }
    public string Phone { get; set; }
    public string Address { get; set; }
    public DateTime CreatedOn { get; set; }
    public DateTime? UpdatedOn { get; set; }
    public DateTime? DeletedOn { get; set; }
    public bool IsDeleted { get; set; }
}